spec/parsers/log-parser-spec.js

```diff
@@ -40,18 +40,18 @@ describe('LogParser', () => {
       const parser = new LogParser(logFile, texFile)
       const result = parser.parse()
 
-      expect(_.countBy(result.messages, 'type').Error).toBe(3)
+      expect(_.countBy(result.messages, 'type').error).toBe(3)
     })
 
     it('associates an error with a file path, line number, and message', () => {
       const logFile = path.join(fixturesPath, 'errors.log')
       const texFile = path.join(fixturesPath, 'errors.tex')
       const parser = new LogParser(logFile, texFile)
       const result = parser.parse()
-      const error = _.find(result.messages, (message) => { return message.type === 'Error' })
+      const error = _.find(result.messages, (message) => { return message.type === 'error' })
```

Comment:
Since we're changing this anyway, maybe we can change this to `Array.find`? But we can also do that in a separate PR at some point (remove as much lodash stuff as possible).